models/base_model.py

```diff
@@ -135,6 +136,17 @@ def get_current_visuals(self):
                 visual_ret[name] = getattr(self, name)
         return visual_ret
 
+    def get_current_visuals_sample(self):
```

Comment:
Update doc to write what it does - why it's different from `get_current_visuals()`